src/pages-and-resources/messages.js

```diff
@@ -5,10 +5,6 @@ const messages = defineMessages({
     id: 'course-authoring.pages-resources.heading',
     defaultMessage: 'Pages & Resources',
   },
-  'enable.button': {
```

Comment:
can you state why this was removed?

Reply:
We are not using `enable button` in the page and resources section so this message is no more needed that's why I removed it.